src/main/java/frc/robot/commands/ShooterPrime.java

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.ShooterSubsystem;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ShooterPrime extends CommandBase {
+  private ShooterSubsystem m_ShooterSubsystem;
+
+  /** Creates a new ShooterPrimary. */
+  public ShooterPrime(ShooterSubsystem shooterSubsystem) {
+    addRequirements(shooterSubsystem);
+    m_ShooterSubsystem = shooterSubsystem;
+
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    m_ShooterSubsystem.prime();
+
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
+    if (!interrupted){
+      //TODO: uncomment when cargoMotor exists 
+      //m_ShooterSubsystem.shoot()
+    }
+    m_ShooterSubsystem.windFlywheel(0); //TODO: Look at this again, we don't want our flywheel to instantly switch off in the end
```

Comment:
Does it make sense to keep the flywheel coasting?  Then you could turn the flywheel off, and let it coast until you need to shoot again.

Reply:
It would, but can I do that? Maybe I can just add voltage control to shooterSubsystem if rpm == 0, so it doesn't actively try to get to 0? Is there a better way to do that?

Reply:
I would recommend opening an issue to track it, and go ahead and close this comment.  I can poke into REV's example code, and post links in the issue.  

I mean, you could change it to m_ShooterSubsystem.flywheelOff() or something, and then the subsystem code can do the right thing. 

But that can be a later action.

Reply:
I just don't know how to let the wheel coast, it slows down pretty quickly. I will add a simple voltage control, so that the motor isn't slowing things down, and is simply just getting no power.